docker/include/populate-m2-repo.sh

```diff
@@ -21,9 +21,12 @@ if [[ -n $(find /opt/rh -mindepth 1 -maxdepth 1 -type d -name "rh-maven*") ]]; t
   source /opt/rh/rh-maven*/enable
 fi
 
+readonly MVNW=/opt/vespa-systemtests/tests/mvnw
```

Comment:
How do you know that this location is writable by the user ? And I dont see how we end up in this dir. I propose to use a temp location and remove the wrapper afterwards.